### ISSUE TITLE
Off mode

### ISSUE THREAD

**Issue by amhenikoff** [phultquist/smart-album-cover#31]
turns off the display when ambient light is below a set threshold for night-time listening and to turn off clock mode at night

**Comment by phultquist** [comment]
doesn't it already turn off at night? this seems like a stupid feature tbh.

**Comment by phultquist** [comment]
i am confused?

**Comment by amhenikoff** [comment]
It only turns off at night if you don't have music playing, and I want a clock mode too.

**Comment by phultquist** [comment]
so what's the point? why did you mention clock mode.

if you have music playing and you... don't want the display on? I'll have a button to turn off the display without unplugging it, but having this seems like an unnecessary way to complicate things

**Comment by amhenikoff** [comment]
I mentioned clock mode because the clock would be lit up, even when music isn't playing.

If you are trying to fall asleep with music playing, or if you like to have clock mode enabled while music isn't playing, it would be nice to have the ability for the panel to turn off automatically, so you don't have to push a button every time you go to bed.

**Comment by phultquist** [comment]
this certainly **will not** be a default feature.

**Comment by phultquist** [comment]
Let's rename this... off mode? sounds like a 3 year old chose two words out of a jar.

"Autosleep"

How about that. But "off mode??? " Seriously????

**Comment by phultquist** [comment]
also it doesn't make sense to do based on a threshold in the room. it should be time based. that's how everything on the phone is done. like do not disturb.

**Comment by amhenikoff** [comment]
Agreed, but nor will clock mode, so I think it's reasonable. It's a tiny bit of code for an in my opinion, very useful feature. Also, I think that no matter how good the app is, the user should have to use it as little as possible, so any sort of automation makes the product more convenient to use, and convenience is so ridiculously important in how much a product gets used. If I had to turn it off every time I wanted to sleep, I would eventually end up leaving it off because it's not worth the hassle of turning it on every morning just to see some pretty lights.

**Comment by amhenikoff** [comment]
> also it doesn't make sense to do based on a light threshold in the room. it should be time based. that's how everything on the phone is done. like do not disturb.

How would it be time-based on the user laying down in bed to sleep. I feel like the only constant in that situation is lights off.
